cmd/metrics-server/app/start.go

```diff
@@ -161,6 +161,8 @@ func (o MetricsServerOptions) Run(stopCh <-chan struct{}) error {
 	if err != nil {
 		return fmt.Errorf("unable to construct lister client config: %v", err)
 	}
+	// Use protobufs for communication with apiserver
+	clientConfig.ContentType = "application/vnd.kubernetes.protobuf"
```

Comment:
This client config is also used when talking to kubelets. Was this intended? Will it work with older kubelet versions?

Reply:
Can you point me to the code where this is used to talk to apiserver? How we do that?
The config determines apiserver...

Reply:
As discussed offline - this is the code that creates config for kubelet client: https://github.com/kubernetes-incubator/metrics-server/blob/b10dfc65239625738c800a2eac6284af6430fa19//pkg/sources/summary/configs.go#L50

This one ignores ContentType - it's basically used only in client-go library when creating a request (which is not the path used to talk to kubelet).

Reply:
Ack, thanks for verifying this!